CHANGES.md

```diff
@@ -1,3 +1,6 @@
+### Master
+- Remove the `infer-timestamp` direction (#171 @Julow)
```

Comment:
I added a CHANGES line.
Should I mark that it's a breaking change ? And, how ?
@NathanReb what do you mean by a "removed" section ?

Reply:
I recenlty updated the changelog, can you rebase on top of that and put that under a `#### Removed` section.

From there I'd say it's pretty clear that removed features are a breaking change.

Reply:
Done